src/devtools/devtools.tsx

```diff
@@ -540,12 +540,24 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
               alignItems: 'center',
             }}
           >
-            <Logo
-              aria-hidden
+            <button
+              type="button"
+              aria-label="Close React Query Devtools"
```

Comment:
I don't think its good to have two buttons with the same `aria-label`. This will also make `findByRole('button', { name: 'Close React Query Devtools' })` error because it will find both buttons.

It's probably best to keep the aria-label here, but remove it from the button that has the `Close` text, as that button is already labelled by the `Close` text printed on it.

Reply:
Removed `aria-label` from Close and kept it in Logo button.